Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:11-jdk-slim as builder
+
+COPY . /home/build/project
+WORKDIR /home/build/project
+RUN ./gradlew build -x test
+
+FROM sonarqube:community
```

Comment:
Shouldn't we allow overriding the sonarqube version here, like in the release docker image?

```
ARG SONARQUBE_VERSION

FROM sonarqube:${SONARQUBE_VERSION}

```

The docker-compose.yml already sets the SONARQUBE_VERSION arg, so would support this change.